applications/CompressiblePotentialFlowApplication/custom_processes/define_2d_wake_process.cpp

```diff
@@ -318,4 +323,25 @@ const BoundedVector<double, 3> Define2DWakeProcess::ComputeDistanceFromTrailingE
     return distance_to_point;
 }
 
+// This function checks that the wake condition is fulfilled up to a certain tolerance
+void Define2DWakeProcess::ExecuteFinalizeSolutionStep()
```

Comment:
Do you need this one defined here as well? Can you call the process directly?

Reply:
Yes you are right. I will move the process to a utility and call it directly from here.